base/gmp.jl

```diff
@@ -458,7 +475,7 @@ powermod(x::Integer, p::Integer, m::BigInt) = powermod(big(x), big(p), m)
 
 function gcdx(a::BigInt, b::BigInt)
     if b == 0 # shortcut this to ensure consistent results with gcdx(a,b)
-        return a < 0 ? (-a,-one(BigInt),zero(BigInt)) : (a,one(BigInt),zero(BigInt))
+        return a < 0 ? (-a,-ONE,ZERO) : (a,ONE,ZERO)
```

Comment:
I'm not sure what to think about this, as mutation of `BigInt` is not covered by the public API. For reference, there was a discussion about "alias-stability" in another [issue](https://github.com/JuliaLang/julia/pull/15546).